pytket-examples-book/maintained-notebooks.txt

```diff
@@ -1,18 +1,17 @@
 ansatz_sequence_example
-backends_example
```

Comment:
I think its because these are the ones that caused trouble with building the jupyterbook. However I don't think its needed to remove them from this list, We can still maintian them. We could also resolve the issues so that they can be executed on the CI with jupyterbook in future.

Reply:
I've added the deleted files back in